op-devstack/presets/interop.go

```diff
@@ -50,17 +50,9 @@ func (s *SimpleInterop) L2Networks() []*dsl.L2Network {
 	}
 }
 
-// startInProcessSimpleInterop starts a new system that meets the simple interop criteria
-func startInProcessSimpleInterop() stack.Option[*sysgo.Orchestrator] {
-	var ids sysgo.DefaultInteropSystemIDs
-	return sysgo.DefaultInteropSystem(&ids)
-}
-
-func ConfigureSimpleInterop() stack.CommonOption {
-	if globalBackend == SysGo {
-		return stack.MakeCommon(startInProcessSimpleInterop())
-	}
-	return nil
+// WithSimpleInterop specifies a system that meets the SimpleInterop criteria.
+func WithSimpleInterop() stack.CommonOption {
+	return stack.MakeCommon(sysgo.DefaultInteropSystem(&sysgo.DefaultInteropSystemIDs{}))
```

Comment:
From my understanding, this change causes devstack to always spawn the sysgo (in-memory) system, even when targeting sysext. Previously, ConfigureSimpleInterop handled this by preventing a call to DefaultInteropSystem when sysext was targeted.

Since ConfigureSimpleInterop is now removed, I believe the sysext targeting path may be broken - although I haven't tested it yet.

Can you confirm if this was considered? If not, this might introduce unintended behavior.